clamscan/clamscan.c

```diff
@@ -184,6 +185,10 @@ int main(int argc, char **argv)
         rmb = info.rblocks * (CL_COUNT_PRECISION / 1024) / 1024.0;
         logg("Data read: %2.2lf MB (ratio %.2f:1)\n", rmb, info.rblocks ? (double)info.blocks / (double)info.rblocks : 0);
         logg("Time: %u.%3.3u sec (%u m %u s)\n", ds, dms / 1000, ds / 60, ds % 60);
+        strftime(buffer, 26, "%Y:%m:%d %H:%M:%S", localtime(&t1.tv_sec));
```

Comment:
It would be better to use sizeof(buffer) here (and in the line below) rather than `26`, just in case there is need to change the size of `buffer`.

Reply:
You're right !